napari/layers/points/points.py

```diff
@@ -623,11 +623,7 @@ def current_properties(self) -> Dict[str, np.ndarray]:
     @current_properties.setter
     def current_properties(self, current_properties):
         update_indices = None
-        if (
-            self._update_properties
-            and len(self.selected_data) > 0
-            and self._mode != Mode.ADD
-        ):
+        if self._update_properties and len(self.selected_data) > 0:
```

Comment:
@kevinyamauchi I don't work much with points properties so not sure if this change makes sense or whether for properties we actually do still want to make it not editable in Add mode?

Reply:
I think it makes sense to update the properties in `Add` mode if we are allowing that for the size, colors, etc., as it makes it consistent. Also, users may be coloring the faces/edges by properties so they will need to update the properties/features to update the color.